.github/workflows/i18n-crowdin-fix-files.yml

```diff
@@ -40,7 +41,27 @@ jobs:
         run: yarn i18n:extract
 
       - name: Commit changes
-        uses: stefanzweifel/git-auto-commit-action@v5
+        uses: stefanzweifel/git-auto-commit-action@8756aa072ef5b4a080af5dc8fef36c5d586e521d # v5.0.0
```

Comment:
Can't we use the latest version of each action instead of a specific commit?

Reply:
pinning to the specific commit was a deliberate decision I made for security. ["Pinning an action to a full length commit SHA is currently the only way to use an action as an immutable release."](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions)